SurveyApp/Models/Survey/SurveyResponse.swift

```diff
@@ -7,7 +7,9 @@
 
 import Foundation
 
-struct SurveyAttributes: Codable {
+protocol SurveyAttributeType {}
```

Comment:
Since there's no common variables in the classes, I think we can jut map to Any?

Reply:
I updated the code. Thank you for a better approach 🙇‍♀️